imgee/storage.py

```diff
@@ -258,9 +268,10 @@ def resize_img(src, dest, size, mimetype, format, is_thumbnail):
 
 def clean_local_cache(expiry=24):
     """
-    Remove files from local cache which are NOT accessed in the last `expiry` hours.
+    Remove files from local cache
+    which are NOT accessed in the last `expiry` hours.
     """
-    cache_path = app.config.get('UPLOADED_FILES_DEST')
+    cache_path = os.path.join(app.static_folder, app.config.get('UPLOADED_FILES_DIR'))
```

Comment:
`os.path.join` will break out of `app.static_folder` if the config setting has `..` or `/` in it. I recall there's a secure version of `path.join` that checks for this. Investigate it? (Not particularly serious though since it's only in server-side config.)